ferrilata-devilline/Properties/launchSettings.json

```diff
@@ -19,6 +19,11 @@
       "commandName": "Project",
       "launchBrowser": true,
       "environmentVariables": {
+        "DB": "server=localhost;database=DevillineDB;user=root;password=",
+        "PASSWORD": "",
+        "HOST": "localhost",
+        "USERNAME": "root",
+        "DATABASE": "DevillneDB",
```

Comment:
Why are there DATABASE & DB at the same time? Also, check the database name - it doesn't match in these two variables.

Reply:
fixed

Reply:
these are obsolete, in case of mac we need to run db migration in terminal, so env vars shoul be set in terminal with sudo nano ~/.bash_profile